cupy/fft/fft.py

```diff
@@ -401,6 +402,9 @@ def _fftn(a, s, axes, norm, direction, value_type='C2C', order='A', plan=None,
     elif order == 'F' and not a.flags.f_contiguous:
         a = cupy.asfortranarray(a)
 
+    # sort the provided axes in ascending order
+    axes = tuple(sorted(np.mod(axes, a.ndim)))
```

Comment:
It seems fixing a bug to move the line after calling `_cook_shape`.  Then I'd like to backport this part to v6.  Is it possible to add a testcase to `tests/cupy_tests/fft_tests/test_fft.py` and detect the issue?

Reply:
Added a test case to `TestFftn` with unsorted axes arguments.

Reply:
Thanks.  I confirmed it.